### PR TITLE
Fix JSON errors in 762x25

### PR DESCRIPTION
**Pull request by mugling**
Self-merge to fix automated builds as I missed some JSON errors in the rebased version of #16810

### CODE REVIEW

File: data/json/itemgroups/magazines.json [hunk 1]
```diff
@@ -141,7 +141,6 @@
       ["calicomag", 15],
       ["fnp90mag", 15],
       ["mp5bigmag", 10],
-      ["ppsh41mag", 10 ],
```
Comment (BorkBorkGoesTheCode) [inline data/json/itemgroups/magazines.json]: That was placed there deliberately. The 35-round magazine is currently being manufactured for collector's edition semiautomatic PPSh variants.

**Comment by BorkBorkGoesTheCode** [comment]
The 35-round magazine is still being manufactured.

File: data/json/itemgroups/magazines.json (hunk 1 above)
Comment (mugling) [inline data/json/itemgroups/magazines.json]: That item wasn't defined so it resulted in JSON load error

**Comment by mugling** [comment]
Your PR still included 35 and 71-round magazines but the naming varied which resulted in JSON errors which I missed when the PR was rebased. Both magazines are still in but the names are updated to fix the error

File: data/json/itemgroups/magazines.json (hunk 1 above)
Comment (narc0tiq) [inline data/json/itemgroups/magazines.json]: Right, but the idea was "put `ppshmag` back in instead of the misnamed one".

Comment (mugling) [inline data/json/itemgroups/magazines.json]: Ah right I see. Those are both obscure magazines though and so presumably better belong in that category?

Comment (BorkBorkGoesTheCode) [inline data/json/itemgroups/magazines.json]: https://en.wikipedia.org/wiki/PPSh-41

Comment (mugling) [inline data/json/itemgroups/magazines.json]: I thought the previous consensus was that item is rare in New England?

**Pull request merged**